docs/general/faq.md

```diff
@@ -12,8 +12,8 @@ We explain our rationale on our [about page](/docs/general/about).
 
 ## Why don't you support my favorite client or feature?
 
-Chances are, we'd like to support it! But it hasn't been implemented yet.
-Jellyfin is an entirely volunteer-driven project, so until a developer is able and willing to implement a feature, it likely won't be done.
+Chances are, we would like to support it! But it has not been implemented yet.
+Jellyfin is an entirely volunteer-driven project, so until a developer is able and willing to implement a feature, it likely will not be done.
```

Comment:
Were these contraction removals done with an automated tool? And is there a particular reason you chose to remove them?

Reply:
> Were these contraction removals done with an automated tool? And is there a particular reason you chose to remove them?

No I manually changed them throughout, it's more formal this way

Reply:
figured it might be something like that and I generally agree - sounds good to me

Reply:
Thanks!